hphp/runtime/vm/jit/abi.cpp

```diff
@@ -77,4 +77,10 @@ RegSet arg_regs_ind_ret(size_t n) {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+uint8_t required_flags(jit::ConditionCode cc) { 
+  return ARCH_SWITCH_CALL(required_flags, cc);
+}
```

Comment:
Initially, the map flags pass was strictly for ARM. I decided to generalize it more, which led to the exposure of this function through the ABI files of the other platforms.